docs/dev/test-configuration-options.md

```diff
@@ -810,6 +810,37 @@ capabilities.setCapability("sauce:options", sauceOptions);
 
 ---
 
+### `customLogFiles`
+<p><small>| OPTIONAL | LIST | <span className="sauceDBlue">Virtual Devices Only</span> |</small></p>
+
+A list of search filters to enumerate after an app test to locate text files to upload as logs. Files are uploaded with the `.log` extension appended. The search paths are rooted at the application under test:
```

Comment:
Can we make this a bit more clear by also explaining why people can use it. For example

> If you app creates extra log then you can use the `customLogFiles` to store those extra logs in the "Logs"-tab of the executed automated session. .....